docker/dependencies/compute_dependencies_hash.sh

```diff
@@ -9,4 +9,4 @@
 # Exit the script at the first failure
 set -e
 
-find dependencies docker/dependencies -not -wholename '*/trilinos_develop/*' -not -name 'README.md' -type f -exec sha1sum {} \; | sort | sha1sum | cut -c -8
+find dependencies/current dependencies/testing docker/dependencies -type f -exec sha1sum {} \; | sort | sha1sum | cut -c -8
```

Comment:
Should we keep excluding a possible README.md? Otherwise, we would need to adapt the script again if we decide to add more documentation. Though, I don't feel super strong about it.
```suggestion
find dependencies/current dependencies/testing docker/dependencies -not -name 'README.md' -type f -exec sha1sum {} \; | sort | sha1sum | cut -c -8
```

Reply:
Well, I would need to adapt the hash now for this ;) I would keep it.

Reply:
Yes. I'm fine if you keep it. But we should keep it in mind.